packages/trader/src/Modules/Reports/Components/indicative-cell.jsx

```diff
@@ -25,7 +25,7 @@ class IndicativeCell extends React.PureComponent {
         return (
             <div className='open-positions__indicative'>
                 <div className='open-positions__indicative--amount'>
-                    <Money amount={Math.abs(amount)} currency={currency} />
+                    <Money amount={Math.abs(amount)} currency={currency} should_format_crypto={false} />
```

Comment:
What happens if `currency={currency}` is of type crypto? We don't want to format it? :thinking:

Reply:
As stated above, here we want to keep original accuracy for currencies (what BE hands over).